iothub/device/src/Common/Exceptions/QuotaExceededException.cs

```diff
@@ -7,8 +7,12 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when an attempt to add a device fails because the maximum number of registered devices has been reached.
+    /// The exception that is thrown by the device client when the daily message quota for the IoT hub is exceeded.
+    /// <seealso href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-troubleshoot-error-403002-iothubquotaexceeded">Troubleshoot Quota Exceeded</seealso>
```

Comment:
Do we want both SeeAlso and the same link in remarks section? Seems redundant to me.

Reply:
Remove the `en-us` path from the URL.

Reply:
Yeah thats fair probably don't need both.